src/ui/components/shared/WorkspaceSettingsModal/WorkspaceSubscription.tsx

```diff
@@ -40,48 +60,52 @@ export default function WorkspaceSubscription({ workspaceId }: { workspaceId: st
   }
 
   return (
-    <>
+    <section className="space-y-6 overflow-y-auto" style={{ marginRight: -16, paddingRight: 16 }}>
       <SettingsHeader>{getViewTitle(view)}</SettingsHeader>
```

Comment:
We should show the trial header: https://www.figma.com/file/SSr1ljyzF0lCXTL7BX1Whh/Replay-Design-Doc?node-id=7247%3A24885